### DISCUSSION TITLE
"Missing Native Function `serialized_size`" ??

### DISCUSSION THREAD

**Discussion by ajaythxkur** [aptos-labs/aptos-developer-discussions#382] [General]
``
INTERNAL TEST ERROR: Unexpected Verification Error
│ Test was not expected to error, but it gave a MISSING_DEPENDENCY (code 1021) error with error message: "Missing Native Function `serialized_size`". Error originating in the module 0000000000000000000000000000000000000000000000000000000000000001::bcs rooted here
``

**Comment by gregnazario** [comment]
Please update to Aptos CLI 4.0.0, and that should build correctly.

The previous versions of the CLI are missing the native function `serialized_size`
